.github/workflows/nodejs-16.yml

```
@@ -0,0 +1,23 @@
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '16.x'
+      - run: npm install
+      - run: npm run prepublishOnly
+      - run: node -v
+      - run: npm run test-node16 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)